lib/commands/web.js

```diff
@@ -22,6 +22,11 @@ const NOTCHED_DEVICE_SIZES = [
   {w: 1125, h: 2436}, // 11 Pro, X, Xs
   {w: 828, h: 1792}, // 11, Xr
   {w: 1242, h: 2688}, // 11 Pro Max, Xs Max
+  {w: 1080, h: 2340}, // 13 mini, 12 mini
+  {w: 1170, h: 2532}, // 14, 13, 13 Pro, 12, 12 Pro
+  {w: 1284, h: 2778}, // 14 Plus, 13 Pro Max, 12 Pro Max
+  {w: 1179, h: 2556}, // 14 Pro
+  {w: 1290, h: 2796}, // 14 Pro Max
```

Comment:
I think we need this to fix `nativeWebTapStrict` feature for these devices.

Reply:
for sure: https://github.com/appium/appium/issues/17014